android/src/app/organicmaps/MapFragment.java

```diff
@@ -1,251 +1,75 @@
-                      implements View.OnTouchListener,
-                                 SurfaceHolder.Callback
+    implements View.OnTouchListener,
+    SurfaceHolder.Callback
 {
-  public static final String ARG_LAUNCH_BY_DEEP_LINK = "launch_by_deep_link";
   private static final String TAG = MapFragment.class.getSimpleName();
+  private final Map mMap = new Map();
 
-  // Should correspond to android::MultiTouchAction from Framework.cpp
-  private static final int NATIVE_ACTION_UP = 0x01;
-  private static final int NATIVE_ACTION_DOWN = 0x02;
-  private static final int NATIVE_ACTION_MOVE = 0x03;
-  private static final int NATIVE_ACTION_CANCEL = 0x04;
-
-  // Should correspond to gui::EWidget from skin.hpp
-  private static final int WIDGET_RULER = 0x01;
-  private static final int WIDGET_COMPASS = 0x02;
-  private static final int WIDGET_COPYRIGHT = 0x04;
-  private static final int WIDGET_SCALE_FPS_LABEL = 0x08;
-
-  // Should correspond to dp::Anchor from drape_global.hpp
-  private static final int ANCHOR_CENTER = 0x00;
-  private static final int ANCHOR_LEFT = 0x01;
-  private static final int ANCHOR_RIGHT = (ANCHOR_LEFT << 1);
-  private static final int ANCHOR_TOP = (ANCHOR_RIGHT << 1);
-  private static final int ANCHOR_BOTTOM = (ANCHOR_TOP << 1);
-  private static final int ANCHOR_LEFT_TOP = (ANCHOR_LEFT | ANCHOR_TOP);
-  private static final int ANCHOR_RIGHT_TOP = (ANCHOR_RIGHT | ANCHOR_TOP);
-  private static final int ANCHOR_LEFT_BOTTOM = (ANCHOR_LEFT | ANCHOR_BOTTOM);
-  private static final int ANCHOR_RIGHT_BOTTOM = (ANCHOR_RIGHT | ANCHOR_BOTTOM);
-
-  // Should correspond to df::TouchEvent::INVALID_MASKED_POINTER from user_event_stream.cpp
-  private static final int INVALID_POINTER_MASK = 0xFF;
-  private static final int INVALID_TOUCH_ID = -1;
-
-  private int mCurrentCompassOffsetX;
-  private int mCurrentCompassOffsetY;
-  private int mBottomWidgetOffsetX;
-  private int mBottomWidgetOffsetY;
-
-  private int mHeight;
-  private int mWidth;
-  private boolean mRequireResize;
-  private boolean mSurfaceCreated;
-  private boolean mSurfaceAttached;
-  private boolean mLaunchByDeepLink;
-  @Nullable
-  private String mUiThemeOnPause;
-  @SuppressWarnings("NullableProblems")
-  @NonNull
-  private SurfaceView mSurfaceView;
-  @Nullable
-  private MapRenderingListener mMapRenderingListener;
-
-  private void setupWidgets(int width, int height)
-  {
-    mHeight = height;
-    mWidth = width;
-    Context context = requireContext();
-
-    nativeCleanWidgets();
-    setupBottomWidgetsOffset(mBottomWidgetOffsetY, mBottomWidgetOffsetX);
-
-    nativeSetupWidget(WIDGET_SCALE_FPS_LABEL,
-                      UiUtils.dimen(context, R.dimen.margin_base),
-                      UiUtils.dimen(context, R.dimen.margin_base),
-                      ANCHOR_LEFT_TOP);
-
-    setupCompass(mCurrentCompassOffsetY, mCurrentCompassOffsetX, false);
-  }
-
-  /**
-   * Moves the map compass using the given offsets.
```

Comment:
Did you intentionally remove comments here and in other places??

Reply:
I moved them to `Map.java`